scout/adapter/mongo/panel.py

```diff
@@ -514,9 +520,11 @@ def apply_pending(self, panel_obj, version):
 
         return inserted_id
 
-    def latest_panels(self, institute_id):
+    def latest_panels(self, institute_id, include_hidden=False):
```

Comment:
I think also here would be best if include_hidden=True is used as default. And then you call the function providing include_hidden=False (as I see you do, from the views)